bin/phpunit

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-docker compose run --rm -u $(id -u):$(id -g) php ./vendor/bin/phpunit ./src \
+docker compose run --rm -u $(id -u):$(id -g) php ./vendor/bin/phpunit ./src/Validators \
```

Comment:
This might be too restrictive if we want to test outside of the Validators dir.

Reply:
Reverted, didn't realize it would traverse, thought it was targeted.